pyproject.toml

```diff
@@ -116,10 +116,16 @@ exclude = [
     "**/tests/conftest.py",
     "**/tests/fauxmplementations/miniatures.py",
     "**/tests/test_*.py",
+    "**/bowtie/benchmarks/*",
+    "**/tests/benchmarks/*",
 ]
 
 [tool.ruff]
 line-length = 79
+exclude = [
```

Comment:
Any reason not to let ruff format these? Perhaps just add specific lint rule exclusions like we do below for tests?

Reply:
I was thinking that these are anyways static files so wont be of much benefit as well, some errors were cropping up. Ig I'll exclude the non useful ones.